server/historian/packages/historian-base/src/routes/summaries.ts

```diff
@@ -100,31 +103,44 @@ export function create(
 		params: IWholeSummaryPayload,
 		initial?: boolean,
 	): Promise<IWriteSummaryResponse> {
-		const service = await utils.createGitService(
+		const service = await utils.createGitService({
 			config,
 			tenantId,
 			authorization,
 			tenantService,
+			storageNameProvider,
 			cache,
 			asyncLocalStorage,
-		);
-		return service.createSummary(params, initial);
+			initialUpload: initial,
+		});
+		const result = await service.createSummary(params, initial);
```

Comment:
>initial [](http://example.com/codeflow?start=53&length=7)

We already passed `initial` to `utils.createGitService()`, why do we pass it again?